pkg/minishift/openshift/service_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openshift
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	test "github.com/minishift/minishift/pkg/testing"
+
+	instanceState "github.com/minishift/minishift/pkg/minishift/config"
+)
+
+func Test_getProject(t *testing.T) {
+	setup()
+	defer teardown()
+
+	fakeRunner, _, cmdToOutput := test.NewFakeRunner()
```

Comment:
I think a more explicit way of configuring the fake runner would be more transparent . Right now you returns two maps which it is hard to know what to do with, especially since there are no godocs either. 

Why not:

    fakeRunner := test.NewFakeRunner(t)
    fakeRunner.expectAndReturn("my 1st command", "my 1st expected response")
    fakeRunner.expectAndReturn("my 2nd command", "my 2nd expected response")
     // ...
 
     // make actual test call
     got, err := getProjects()

     // verify what we got is correct

     // verify all expected calls in the right order where received by the FakeRunner
     fakeRunner.verify()